source/contributing/documentation/diataxis.md

```diff
@@ -39,13 +39,13 @@ A concept can also describe the historical context behind why something works th
 
 If you find yourself wanting to write about the nitty gritty details of how something works, you most likely want to write an explanation.
 
-### Recipes vs. Tutorials
+### Guide vs. Tutorials
```

Comment:
Maybe make Guide also plural or conversely, make Tutorial a singular (pedantic changes just to keep consistency)

Reply:
```suggestion
### Guides vs. Tutorials
```

Yes, correct, thanks